packages/digitalocean-js/src/lib/services/image/image.service.ts

```diff
@@ -1,6 +1,6 @@
-import { Action, Image } from '../../models';
```

Comment:
Looks like you've got several autoformatted quote changes in here. Could you make sure those stay single quote? If you use VSCode with auto format on save, make sure you have the Prettier extension installed and it should respect the repo's `.prettierrc` configuration.